api/npm.ts

```diff
@@ -163,24 +162,24 @@ const download = async (period: string, npmName: string, tag = 'latest') => {
   }
 }
 
-// https://github.com/astur/check-npm-dependents/blob/master/index.js
```

Comment:
Not relevant anymore since I'm not scraping HTML but hydration data instead (JSON).